.github/workflows/test.yaml

```
@@ -144,7 +144,7 @@ jobs:
         run: |
           conda activate test-environment
           conda uninstall panel bokeh --force --offline -y || true
-          pip install bokeh panel --pre
+          pip install bokeh 'panel==1.4.0b2' --pre
```

```suggestion
          pip install bokeh panel --pre
```

This will fixed in the next dev release in Panel